### DISCUSSION TITLE
Deep Learning 簡介 / 從人腦啟發的 Deep Learning

### DISCUSSION THREAD

**Discussion by minniewei** [1101-GS4719-C/Python-Community-GS4719-C#494] [General]
我之前曾經用到tensorflow並產生了興趣。tensorflow是一個用來進行deep learning的框架，其背後的原理是內神經網路。雖然在這個網路發達的時代我們能直接從線上抓取此框架並完成許多的事情，但我認為還是有必要去學習內神經網路的原理。透過以下我分享的這篇文章我們能有效了解其概念以及常見機器學習的幾種方式，希望各位能學習到東西!!!
https://ithelp.ithome.com.tw/articles/10237540

**Comment by ThomasMeng0515** [comment]
機器學習若從原理來講可以是一門很深的學問，但是我們在實作的時候感覺都是直接套用模型更改參數，所以確實需要再深入理解其原理，才能了解該如何修改模型以獲得更好的結果

**Comment by xkllkx** [comment]
在學習機器學習時，真的需要知道模型背後的原理才能進行優化，不然就真的只能賭運氣土法煉鋼了。

**Comment by sean1325418** [comment]
目前學深度學習有一種練習通靈的感覺，有些參數只能慢慢試才能知道結果。猜中上天堂猜錯就浪費超多時間。

**Comment by jason79461385** [comment]
深度學習參數真的很難調，做專題時教授希望我們可以不用手動調整就能夠產出正確的，真的好難啊。
